src/main/webapp/WEB-INF/tags/instructor/search/commentsForResponsesSearchResults.tag

```diff
@@ -6,13 +6,22 @@
 <br>
 <div class="panel panel-primary">
     <div class="panel-heading">
-        <strong><jsp:doBody/></strong>
+        <strong><jsp:doBody /></strong>
```

Comment:
There shouldn't be a need to change this tag, this tag is for response comments, not comments for students